src/Time/TimeSteppers/AdamsBashforthN.hpp

```diff
@@ -759,9 +759,10 @@ bool AdamsBashforthN::can_change_step_size(
   // evolution until the self-start history has been replaced with
   // "real" values.
   const evolution_less<Time> less{time_id.time_runs_forward()};
-  return history.size() == 0 or
-         (less(history.back(), time_id.step_time()) and
-          std::is_sorted(history.begin(), history.end(), less));
+  return time_id.slab_number() > 0 and
```

Comment:
Should this be `>=`? I'm not sure when we can start allowing. I also remember there being some discussion on an function to check `is_self_starting` but I seem to have yet again forgotten if that got added :P